docs/en_US/Nnictl.md

```diff
@@ -456,6 +477,28 @@ Debug mode will disable version check function in Trialkeeper.
     nnictl experiment import [experiment_id] -f experiment_data.json
     ```
 
+<a name="platform"></a>
+![](https://placehold.it/15/1589F0/000000?text=+) `Manage platform information`
+
+* __nnictl platform clean__
+
+  * Description
+
+    clean up platform data.
```

Comment:
Change it like below,

It uses to clean up disk on a target platform. The provided YAML file includes the information of target platform, and it follows the same schema as the NNI configuration file.

Note, if the target platform is being used by other users, it may cause unexpected errors to others.

Reply:
Fixed.